runtime/src/currency.rs

```diff
@@ -1,3 +1,5 @@
+// TODO: What is the purpose of this?
```

Comment:
No longer used. this was used in original implementation of xcm support.

I will remove it.